descriptors/ModuleDescriptor-template.json

```diff
@@ -5,7 +5,7 @@
   "provides": [
     {
       "id": "calendar",
-      "version": "3.0",
+      "version": "3.1",
```

Comment:
i think this is a breaking change, response was changed from 500 to 422

Reply:
Raise the version to v.4.0